tests/integration/conftest.py

```diff
@@ -41,6 +44,71 @@ async def client(kubeconfig):
     yield client
 
 
+@pytest.fixture(scope="module")
+def subnet_resource(client):
+    return create_global_resource("kubeovn.io", "v1", "Subnet", "subnets")
+
+
+@pytest.fixture(scope="module")
+def worker_node(client):
+    # Returns a worker node
+    for node in client.list(Node):
+        if node.metadata.labels["juju-application"] == "kubernetes-worker":
+            return node
+
+
+@pytest.fixture(scope="module")
+async def gateway_server(ops_test):
+    cmd = "juju run --unit ubuntu/0 -- sudo apt install -y iperf3"
+    rc, stdout, stderr = await ops_test.run(*shlex.split(cmd))
+    assert rc == 0, f"Failed to install iperf3: {(stdout or stderr).strip()}"
```

Comment:
this is dumb, but you could use `ops_test.juju` 

```suggestion
    cmd = "run --unit ubuntu/0 -- sudo apt install -y iperf3"
    rc, stdout, stderr = await ops_test.juju(*shlex.split(cmd))
    assert rc == 0, f"Failed to install iperf3: {(stdout or stderr).strip()}"
```